### PR TITLE
doc: improve/revisit `:h clipboard-x11`

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: runtime/doc/provider.txt [hunk 1]
```diff
@@ -162,13 +162,14 @@ process has not died, the cached selection is applied.
...
-The contents of selections are held by the originating application (e.g., upon
-a copy), and only passed on to another application when that other application
-asks for them (e.g., upon a paste).
+The clipboard providers for X11 store text in "selections".
+Selections are "owned" by an application, so when the application gets closed,
+the selection text is not available anymore.  Clipboard managers can help to
+keep selections nonetheless.
+
+Applications hold the contents of selections  (e.g., upon a copy), and pass
+them on to other application when that other application asks for them (e.g.,
```
Comment (justinmk) [inline runtime/doc/provider.txt]: "other applications" ?

**Comment by justinmk** [comment]
Doesn't seem significantly clearer than the old text. Maybe just add the mention of "Clipboard managers".

**Comment by blueyed** [comment]
It is also changed to use active voice.
Feel free to take what you like from it, or just close it.

**Comment by justinmk** [comment]
@blueyed what's an example of a clipboard "manager"?

**Comment by bfredl** [comment]
Also some application fork themselves without needing a manager. GTK has the support, but IIRC the individual application has to invoke it.

**Comment by blueyed** [comment]
@justinmk 
I am using CopyQ (https://github.com/hluk/CopyQ), but there are a lot, e.g. https://github.com/CristianHenzel/ClipIt.
https://wiki.archlinux.org/index.php/Clipboard#List_of_clipboard_managers

**Comment by justinmk** [comment]
merged into #7188 . I left out the mention of clipboard managers, it's hard to say anything meaningful about them. Yes, third party software exists for all kinds of things. But unless we can make a concrete recommendation it doesn't help readers of the documentation to mention this.

**Comment by blueyed** [comment]
Not pushed yet?

**Pull request closed without merging**